src/api/resources/openApi.json

```diff
@@ -367,12 +367,53 @@
               "properties": {
                 "description": "Additional feature properties",
                 "type": "object",
-                "additionalProperties": true
+                "additionalProperties": true,
+                "properties": {
+                  "points": {
```

Comment:
I think this property name should reflect the fact that this is _metadata for the points_ and should go with the `geometry` LineString meta . Can't think of anything terribly clever, how about `geometryMeta` or `LineStringMeta`?

Reply:
`coordinatesMeta`?

Reply:
Works for me.